reference-architecture/osp-cli/ch5.3_control_network.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# Set OCP3_DNS_NAMESERVER to override
+OCP3_DNS_NAMESERVER=${OCP3_DNS_NAMESERVER:-8.8.8.8}
+PUBLIC_NETWORK=${PUBLIC_NETWORK:-public_network}
+openstack network create control-network 
+openstack subnet create --network control-network --subnet-range 172.18.10.0/24 \
```

Comment:
I'd say to avoid hardcoding the network IPs and create variables with default values instead